### PR TITLE
#706 : Solved a few problems with activities.html

### PR DESCRIPTION
**Pull request by nzec**
- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

* Added activities to nav bar.
* Changed the button text to activities.
* Changed the button's link.
* Changed the placeholder text of the search bar.
* Solves 1st, 2nd 3rd point of #706 

***Preview Link : https://duttaditya18.github.io/gci17.fossasia.org/activities***

### CODE REVIEW

File: js/contributors.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ $(document).ready(function(){
       $(".issue-container").append("<li class='mi'>Sorry there are no issues labeled feature currently opened at gci17.fossasia.org</li>");
     }
 
-    $(".issue-container").append("<a href='https://github.com/fossasia/gci17.fossasia.org/issues' class='btn totalIssue'>" + allIssues.length + " issues on <i class='fa fa-github'></i> Github");
+    $(".issue-container").append("<a href='https://github.com/fossasia/gci17.fossasia.org/issues?utf8=%E2%9C%93&q=is%3Aopen+' class='btn totalIssue'>" + allIssues.length + " activities on <i class='fa fa-github'></i> Github");
```
Comment (harshit98) [inline js/contributors.js]: Let the link remain same: `href=https://github.com/fossasia/gci17.fossasia.org/issues`

Comment (nzec) [reply js/contributors.js]: But then, it only shows the issues and not all activities.

Comment (nzec) [reply js/contributors.js]: All right, I'll change that.

Comment (nzec) [reply js/contributors.js]: @harshit98 Done!

**Comment by nzec** [comment]
@harshit98 Done!

**Review by harshit98 — APPROVED** [review]
LGTM 💯 

@duttaditya18 Good work! The pull request will be merged once Travis ci passes.

**Comment by nzec** [comment]
@harshit98 The Travis build has passed!

**Review by 0x48piraj — APPROVED** [review]

**Pull request merged**